bin/boring-cli.js

```diff
@@ -25,6 +25,7 @@ require('yargs')
     .command('up', 'Run `docker-compose up` to launch infrastructure such as reverse proxy', makeCmd('docker-compose-up'))
     .command('down', 'Run `docker-compose down` to tear down infrastructure', makeCmd('docker-compose-down'))
     .command('yo', 'Generate routes / views / app structure', makeCmd('yo'))
+    .command('generate', 'Alias to `yo`', makeCmd('yo'))
```

Comment:
so the two "new" commands people can use are

`npx boring yo`
`npx boring generate`

Both do the same thing